crates/sui-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -3,10 +3,16 @@ source: crates/sui-config/tests/snapshot_tests.rs
 expression: network_config
 ---
 validator_configs:
-  - protocol-key-pair: VTDx4HjVmRBqdqBWg2zN+zcFE20io3CrBchGy/iV1lqZ8l72H4AyuRRjZGCYLFzG8TTvHdrnZlfyy/7B6/yNCXN0CA32/PDcuLxLDY4K9dgOu/8rTFmfVPQtYxLfwxQnYHjBzDR+u77FGYviWFE/OGuTDQLCdJqAPiMwlV69GhA=
-    worker-key-pair: fsz863qJR38mPjuvloaZBE4vbibFPgrwQXUa+OGTTNMfKnkBqELk02I7xhzhwXF9LnaOD9ct+DQa9WYVuRbwOA==
-    account-key-pair: AJTkqEkxROnmliCMvtACWebdtW+Xm5HDKWdWCYW+YZCbZeUwqtiSP4IuiO1U/Br2sTVBQpkckD5caF+BanumitE=
-    network-key-pair: KUrb+C2JL5zKk+XaKXpgLj08mCQegCCekpnR0SLAhTC+ODDLDFddhaGYhOUoOWrDb5Dh27EywDr6A1haMSoZgA==
+  - protocol-key-pair:
+      value:
+        name: mfJe9h+AMrkUY2RgmCxcxvE07x3a52ZX8sv+wev8jQlzdAgN9vzw3Li8Sw2OCvXYDrv/K0xZn1T0LWMS38MUJ2B4wcw0fru+xRmL4lhRPzhrkw0CwnSagD4jMJVevRoQ
+        secret: VTDx4HjVmRBqdqBWg2zN+zcFE20io3CrBchGy/iV1lo=
```

Comment:
note that the serialization for `protocol-key-pair` looks different than the rest three that uses `SuiKeyPair`. this is because it uses the derived serializer from fastcrypto. this will be resolved when we add BLS to `SuiKeyPair`.

this is not concerning because in production keypairs are using the file path option. the `NetworkConfig` serialization is only used for swarm/testing that contains in-place serialized keypairs.

Reply:
will be fixed by https://github.com/MystenLabs/fastcrypto/pull/324